helpers/test/test_utils.py

```diff
@@ -14,14 +14,15 @@ def generate_test_assets(account, path, amount):
 
     for address in path:
         asset = interface.IERC20(address)
+        asset.approve(router.address, 0, {"from": account})
         asset.approve(router.address, MaxUint256, {"from": account})
 
     # Buy path[n-1] token with "amount" of path[0] token
     router.swapExactETHForTokens(
         0,
         path,
         account,
-        int(time.time()) + 1200, # Now + 20mins,
+        int(time.time()) + 120000000, # Now + 20mins,
```

Comment:
fix comment

Reply:
It doesn't work when the test runs for too long that's why I changed to a bunch of 0s